vuepress/docs/v7.0/docs/reference/deployment-structure.md

```diff
@@ -4,154 +4,128 @@ sidebarDepth: 2
 
 # Entando Deployment Structure
 
-This page provides a high level overview of the key Entando GitHub repositories along with a brief description 
-of how those repositories are realized in a running Entando deployment. The descriptions provided here are meant 
-as a guide for identifying opportunities to dig deeper into the architecture and how things are structured 
-rather than a panacea for understanding the architecture.
+This page provides an overview of the key Entando GitHub repositories with a brief description 
+of how these repositories are realized in a running Entando deployment. It also explores the architecture behind Entando and how its functions are structured.
```

Comment:
"realized" is an awkward word choice. created, implemented, etc.

"behind Entando" and "its functions" are misleading; not "Entando" unless you're equating Entando to the Entando Platform. "...behind the Entando Platform..."